Shakikka/rotten-tomatillos
# Features/display movies

## What does this PR do?

- Creates a basic grid layout of all of the movie posters found in the given data

## How should it be tested in the UI?

- Opening the page with npm start, scrolling to see all of the movie posters to make sure they are appearing as expected
